### PR TITLE
Branding changes

### PR DESCRIPTION
**Pull request by phenning**
@rustd 

Updated the usersecretsId as well to remove aspnet5 from the id pattern

### CODE REVIEW

**Comment by rustd** [comment]
:shipit: 
As a followup should we change the vs template file names from DNX to DotNet or something else to match the grouping parameter @joeloff is using for Item templates. We would only do this if this is something a customer would see.

**Comment by joeloff** [comment]
Customers don't typically see these file names as they live in the VS install folders so it shouldn't be a customer facing issue I think

**Comment by phenning** [comment]
I think this is something we could change for RTW though, if we wanted to.
- Phil

On Jan 27, 2016, at 6:52 PM, joeloff <notifications@github.com<mailto:notifications@github.com>> wrote:

Customers don't typically see these file names as they live in the VS install folders so it shouldn't be a customer facing issue I think

##

**Comment by rustd** [comment]
I see. This is fine then.
/cc @Eilon JFYI

File: src/BaseTemplates/DNXEmptyWeb.vstemplate [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <VSTemplate Version="3.0.0" Type="ProjectGroup" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" xmlns:sdk="http://schemas.microsoft.com/developer/vstemplate-sdkextension/2010">
   <TemplateData>
-    <Name>ASP.NET 5 Empty Web Application</Name>
-    <Description>A project for creating an empty ASP.NET 5 web application</Description>
+    <Name>ASP.NET Core 1 Empty Web Application</Name>
```
Comment (Eilon) [inline src/BaseTemplates/DNXEmptyWeb.vstemplate:4]: If these names show up in the New Project dialog then these shouldn't have versions in them.

cc @DamianEdwards

Comment (rustd) [inline src/BaseTemplates/DNXEmptyWeb.vstemplate:4]: They will show up as description for the template as shown below.

oneaspnet

Comment (Eilon) [inline src/BaseTemplates/DNXEmptyWeb.vstemplate:4]: Yeah I think @DamianEdwards doesn't want versions there... @DamianEdwards can you confirm?

**Pull request merged**